components/UserProvider.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withApollo } from '@apollo/client/react/hoc';
-import jwt from 'jsonwebtoken';
+import jwtDecode from 'jwt-decode';
```

Comment:
Could we use the 'jose' library here too? Or it's not a good idea.

Reply:
Its good, it results in the same gziped size when bundles into _app.